server/routes/visitJourney/dateAndTime.ts

```diff
@@ -99,6 +99,9 @@ export default class DateAndTime {
 
     req.session.slotsList = slotsList
 
+    // set allowOverBooking to false on load of select date and time page
+    req.session.allowOverBooking = false
```

Comment:
if you agree with suggestion to move this inside `visitSessionData` it would instead be set initially in the POST route for prisoner profile (`routes/prisoner.ts`) where `visitSessionData` is defined for a new booking journey.

Reply:
having it also reset on load of date time page - as if returning to this page, could be a different session (so should really re-confirm they want to overbook)

Reply:
👍 good idea - I hadn't thought of this scenario